$OpenBSD: patch-doc_lftp_1,v 1.4 2019/03/10 17:27:01 jca Exp $

- avoid information loss in both groff and mandoc
  upstream commit 57b7098a4e0a7d3e7609ef3b2fb00b500df7a060

Index: doc/lftp.1
--- doc/lftp.1.orig
+++ doc/lftp.1
@@ -788,7 +788,8 @@ FXP is automatically used for transfers between FTP se
 Some FTP servers hide dot-files by default (e.g. \fI.htaccess\fP), and show
 them only when LIST command is used with \-a option. In such case try to use
 `set ftp:list-options \-a'.
-.PP The recursion modes `newer' and `missing' conflict with \-\-scan\-all\-first,
+.PP
+The recursion modes `newer' and `missing' conflict with \-\-scan\-all\-first,
 \-\-depth\-first, \-\-no\-empty\-dirs and setting mirror:no\-empty\-dirs=true.
 
 .B mkdir
